### PR TITLE
DTGB-557: Added template for address fields and fix boxes for detail …

### PR DESCRIPTION
**Pull request by meirege**
…pages.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

File: templates/contrib/address/address-plain.html.twig [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+ *     - country.name: Country name.
+ *
+ * if a subdivision (dependent_locality, locality, administrative_area) was
+ * entered, the array will always have a code. If it's a predefined subdivision,
+ * it will also have a name. The code is always prefered.
+ *
+ * @ingroup themeable
+ */
+#}
+<address class="address" translate="no">
```
Comment (hstandaert) [inline templates/contrib/address/address-plain.html.twig]: Any specific reason why you added the `address` class? I don't see any styling applied to it, nor does it have any added value.

Comment (delrueba) [inline templates/contrib/address/address-plain.html.twig]: W3C: The address tag defines the contact information for the author/owner of a document or an article.
The address tag should NOT be used to describe a postal address, unless it is a part of the contact information.

We can't use it for any old random address, only for website/author contact info.

**Review by delrueba — DISMISSED** [review]

**Review by hstandaert — CHANGES_REQUESTED** [review]

File: templates/contrib/address/address-plain.html.twig (hunk 1 above)
Comment (meirege) [reply templates/contrib/address/address-plain.html.twig]: It's a drupal template. :) I'll delete the class.

Comment (meirege) [reply templates/contrib/address/address-plain.html.twig]: Changed to div.

**Review by hstandaert — APPROVED** [review]

**Pull request merged**